### DISCUSSION TITLE
expand pg_net functionality with more operations and other data types?

### DISCUSSION THREAD

**Discussion by davidaventimiglia-professional** [supabase/supabase#11370] [Feature Requests]
Hello,

The [pg_net](https://github.com/supabase/pg_net) extension for async HTTP calls looks terrific.  Thanks!  Are there any plans do enhance this with more functionality?  

E.g. operations:

- `PUT`
- `PATCH`
- `DELETE`

And other `body` data types:

- `varchar`
- `varchar:bytea`

**Comment by steve-chavez** [comment]
Hello, could you open issues on https://github.com/supabase/pg_net/issues/ for these?

> DELETE

Requested on https://github.com/supabase/pg_net/issues/63

> varchar:bytea

Related https://github.com/supabase/pg_net/pull/27

**Reply by davidaventimiglia-professional** [reply]
You bet.  [Done](https://github.com/supabase/pg_net/issues/77)
